examples/Makefile-nrf52840

```diff
@@ -79,6 +79,10 @@ COMMONCFLAGS                   := \
     -I$(CONFIG_FILE_PATH)         \
     $(NULL)
 
+ifeq ($(CERT_LOG),1)
+COMMONCFLAGS += -DOPENTHREAD_CONFIG_ENABLE_DEFAULT_LOG_OUTPUT=1
```

Comment:
Is what's provided in [examples/common-switches.mk](https://github.com/openthread/openthread/blob/master/examples/common-switches.mk) insufficient for your purposes?